### DISCUSSION TITLE
SimpleGrid how to justifyContent

### DISCUSSION THREAD

**Discussion by francestu96** [chakra-ui/chakra-ui#7916] [Q&A]
I have this mock code which produces the following result

```
<SimpleGrid minChildWidth={["50%", "40%", "30%", "20%"]} spacing='2'>
  {
     properties.map((x: any, i: number) => {
          return (
            <Box p={6} boxShadow={'2xl'} rounded={'lg'} zIndex={1}>
              <Stack justifyContent="space-between">
                <Text>Random</Text>
                <Text>text</Text>
              </Stack>
            </Box>
          )
       })
    }
</SimpleGrid>
```
![image](https://github.com/chakra-ui/chakra-ui/assets/32491001/8c0130da-fdc4-4b39-9253-51a36029853a)

As you can see, if the first Text is longer than the box, it will expand the parent. What I'd like to achieve is to set a **justifyContent="space-between"** in the Stack tag in order to get the following result:
![image](https://github.com/chakra-ui/chakra-ui/assets/32491001/7a2f3d1f-c4dc-4495-8300-01db0cfc2318)

The justifyContent prperty does nothing and I cannot achieve this result

**Comment by aysegulkavakli** [comment]
You should assign a height value for the Stack component, in your code there is no enough space to space between the contents.

```
  <Box p={6} boxShadow={"2xl"} rounded={"lg"} zIndex={1}>
     <Stack justifyContent="space-between" minH="300px">
           <Text>Random</Text>
           <Text>text</Text>
     </Stack>
 </Box>
```
